spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'post_show', type: :feature do
+  before :each do
+    @user1 = User.create(
+      name: 'User 1',
+      photo: 'https://unsplash.com/photos/vpwDZgKtgVk',
+      bio: 'User1 bio',
+      posts_counter: 0
+    )
+    @post1 = Post.create(author: @user1, title: 'post1', text: 'post', comments_counter: 0, likes_counter: 0)
+    @comment1 = Comment.create(author: @user1, post: @post1, text: 'comments')
+    visit user_post_path(@user1, @post1)
+  end
+
+  it 'I can see the posts title' do
+    expect(page).to have_content('post1')
+  end
+
+  it 'I can see who wrote the post' do
+    expect(page).to have_content('User 1')
+  end
+
+  it 'I can see how many comments it has.' do
+    expect(page).to have_content(@post1.comments_counter)
+  end
+
+  it 'I can see how many likes it has' do
+    expect(page).to have_content(@post1.likes_counter)
+  end
+
+  it 'I can see the post body.' do
+    expect(page).to have_content(@post1.text)
+  end
+
+  it 'I can see the name of each commentor.' do
+    expect(page).to have_content(@comment1.author.name)
+  end
+
+  it 'I can see each comment that was left.' do
+    expect(page).to have_content("#{@comment1.author.name}: #{@comment1.text}")
+  end
+end
```

Comment:
- [ ] According to the project requirements, you're expect to write test that ensures correct view template is rendered for both the index and show for the post controller.


```
 expect(response).to render_template(:show)

```

- [ ] Your test cases should include a test  that ensures the response body for the correct placeholder text. 

```
Example :
expect(response.body).to include("Welcome to post show  page")

```

### Project Requirements

![test](https://github.com/Dadadon/blog/assets/26597432/3ace53a6-4fb8-4a5a-81cf-d1a0063995a7)